packages/pwa-kit-extension-sdk/src/configs/utils.ts

```diff
@@ -10,18 +10,24 @@ import dedent from 'dedent'
 import Handlebars from 'handlebars'
 
 // Local
-import {kebabToUpperCamelCase, nameRegex} from '../shared/utils'
+import {kebabToUpperCamelCase} from '../shared/utils'
 
 // Types
 import {ApplicationExtensionsLoaderOptions} from './webpack/types'
 
 // Register Handlebars helpers
 Handlebars.registerHelper('getInstanceName', (aString: string) => {
-    const match = aString.match(nameRegex)
-
     // Explicitly define `namespace` and `name` as strings with fallback values
-    const namespace = match?.[1] ?? ''
-    const name = match?.[2] ?? ''
+    let namespace = ''
+    let name = aString
+
+    // If the package name has a namespace, extract it
+    if (aString.startsWith('@') && aString.includes('/')) {
+        const [namespaceWithAt, packageName] = aString.split('/')
+        // Remove the @ prefix
+        namespace = namespaceWithAt.substring(1)
+        name = packageName
+    }
```

Comment:
We might be able to clean this up a little bit:

```
const hasNamespace = aString.startsWith('@') && aString.includes('/')
const [namespace, name] = hasNamespace
  ? aString.slice(1).split('/')
  : ['', aString]
```

Alternatively could we use an inline regex and retain that type of logic, e.g. a regex that doesn't explicitly have the "extension" string in it